src/copy_tree.rs

```diff
@@ -26,6 +27,7 @@ pub fn copy_tree<ST: ReadTree, DT: WriteTree>(
     mut dest: DT,
     options: &CopyOptions,
 ) -> Result<CopyStats> {
+
```

Comment:
I think `rustfmt` should strip out the blank lines here? Please run `cargo fmt` before committing.